### ISSUE TITLE
How to change graphiQL welcome message?

### ISSUE THREAD

**Issue by thardy** [graphql/express-graphql#447]
I'd like to customize the "# Welcome to GraphiQL..." help block that appears in the editor.  Is this possible?

**Comment by IvanGoncharov** [comment]
@thardy You can customize it in GraphiQL using `defaultQuery ` property:
https://github.com/graphql/graphiql#user-content-usage
But `express-graphql` doesn't expose this option.
Can you please describe your use case?

**Comment by Betree** [comment]
Other configurations options may be useful too like `editorTheme`.

A good way to surface that would be to accept graphiql [props](https://github.com/graphql/graphiql#usage) as a parameter for the `graphiql` config entry:

```es6
app.use('/graphql', graphqlHTTP(async (request, response, graphQLParams) => ({
  schema: MyGraphQLSchema,
  graphiql: {
    editorTheme: 'solarized light',
    defaultQuery: '# Welcome to our GraphiQL server! Please type your commands here'
  }
})));
```

**Comment by geraldoramos** [comment]
@Betree The above example does not work, it will not load the default query. Thoughts?

**Comment by Betree** [comment]
@geraldoramos It's not something that exist today, my example was a proposal of something that may be useful to have in `express-graphql`

**Comment by IvanGoncharov** [comment]
Fixed in #508 and will be included in the next release.
